packages/braid-design-system/project.json

```diff
@@ -0,0 +1,17 @@
+{
+  "$schema": "https://raw.githubusercontent.com/nrwl/nx/15.0.0/packages/nx/schemas/project-schema.json",
+  "targets": {
+    "build": {
+      "dependsOn": ["copy-codemod", "generate-icons"],
+      "executor": "nx:noop"
+    },
+    "copy-codemod": {
+      "dependsOn": ["^build"],
+      "executor": "nx:run-script",
+      "options": {
+        "script": "copy-codemod"
+      }
```

Comment:
Do we need the executor in this case? Isn't the `copy-codemod` script going to be run implicitly once the dependencies are done?

Reply:
That makes sense, but [Nx thinks otherwise](https://nx.dev/packages/nx/executors/run-script#script). The `script` option is required.

Reply:
Lol, nice schema validation nx nerds. Let's leave it as is then 👍